components/script/dom/bindings/settings_stack.rs

```diff
@@ -62,12 +63,17 @@ impl Drop for AutoEntryScript {
         STACK.with(|stack| {
             let mut stack = stack.borrow_mut();
             let entry = stack.pop().unwrap();
-            assert_eq!(&*entry.global as *const GlobalScope as usize,
-                       self.global,
+            assert_eq!(&*entry.global as *const GlobalScope,
+                       &*self.global as *const GlobalScope,
```

Comment:
The compiler complains that &GlobalScope doesn't implement the Debug trait if I get rid of the pointer casts.